src/config.c

```diff
@@ -306,6 +308,17 @@ static struct section_def section_defs[] = {
         MB_FIELD(odd_area_border_color, "#0048", parse_color, noop),
         MB_FIELD(history_border_color, "#3339", parse_color, noop)
     ),
+    SECTION(
+        mode_split, MS_FIELD(pointer_size, "#fffd", parse_color, noop),
```

Comment:
```suggestion
        mode_split,
```

Duplicate and erroneous field definition.

Reply:
Will fix this.